spaceline-all-the-icons-segments.el

```diff
@@ -709,7 +709,7 @@ When FAMILY is provided, put `:family' property into face."
          (family (all-the-icons-icon-family icon))
          (raise (if (> (length (spaceline-all-the-icons-icon-set-flycheck-slim)) 3) -0.2 0.0))
          (icon-face `(:foreground ,(face-foreground face) :height ,(spaceline-all-the-icons--height height))))
-    (when family (plist-put icon-face :family family))
+    (when family (setq icon-face `(append (:family family))))
```

Comment:
Here, the backtick should be before the list, 
_i.e._ ``(append `(:family ,family))``